src/main/java/edu/psu/sweng894/group1/capstone/data/entities/Comment.java

```diff
@@ -16,6 +16,7 @@
      * The parent, optional, of this comment. If this comment is a reply to a comment, this
      * is non-null. If this comment is attached directly to a poll, this is null.
      */
+    // I think this is premature. We don't need to do threading yet and we should add this when it's needed
```

Comment:
This is a tree ID, it has nothing to do with threading, I feel we can leave this in here as when we are ready we can display nested comments using a css proper based on the model value

Reply:
So it does have to do with threading (ie nested comments?). I think we should remove it UNTIL we need it.